src/styles/_variables.scss

```diff
@@ -43,10 +43,26 @@
     --mynah-color-status-warning: #b2911c;
     --mynah-color-status-error: #d91515;
 
+    --mynah-color-status-info-bg: rgba(9, 113, 211, 0.15);
```

Comment:
Instead of duplicating colors with different opacities, it might be better to use pseudo elements for the background and play with its opacity. 

In case if you have to use a dedicated color (not applicable for the informationBlocks status section), it means that they don't need to be exposed as themeable items  and you can generate them through scss functions.